src/Db/Diff/PKSimDB_diff.txt

```diff
@@ -1 +1,30 @@
-INSERT INTO tab_transports(transport_direction,process,source_organ,source_compartment,target_organ,target_compartment,source_organ_type,source_compartment_type,target_organ_type,target_compartment_type) VALUES('InfluxBrainPlasmaToInterstitial','ActiveInfluxSpecificPlasmaToInterstitial','Brain','Plasma','Brain','Interstitial','ORGAN','COMPARTMENT','ORGAN','COMPARTMENT');
+CREATE TABLE tab_disease_states
+(
+    disease_state  text not null
+        constraint tab_disease_states_pk
+            primary key,
+    display_name   text not null,
+    description    text not null,
+    implementation text
+);
+INSERT INTO tab_disease_states(disease_state,display_name,description,implementation) VALUES('CKD','CKD Display TODO','CKD Description TODO','CKDDiseaseState');
+CREATE UNIQUE INDEX tab_disease_states_description_uindex
+    on tab_disease_states (description);
+CREATE UNIQUE INDEX tab_disease_states_disease_state_uindex
+    on tab_disease_states (disease_state);
+CREATE UNIQUE INDEX tab_disease_states_display_name_uindex
+    on tab_disease_states (display_name);
+CREATE TABLE tab_population_disease_states

```

Comment:
@Yuri05 I Am not getting it. 
With DataGrip, I can add whatever I want, update is not respected etc.. It used to work before. Is the definition of the tab_population_disease_states wrong?